subsys/nrf_security/src/drivers/cracen/cracenpsa/src/asymmetric.c

```diff
@@ -43,31 +38,26 @@ cracen_asymmetric_crypt_internal(const psa_key_attributes_t *attributes, const u
 				 enum cipher_operation dir)
 {
 #if PSA_MAX_RSA_KEY_BITS > 0
-	psa_status_t status = PSA_SUCCESS;
-	struct sitask t;
+	psa_status_t status;
 	const struct sxhashalg *hashalg;
-	struct si_rsa_key pubkey;
+	struct cracen_rsa_key pubkey;
 	struct sx_buf modulus;
 	struct sx_buf exponent;
 
 	if (!is_alg_supported(alg)) {
 		return PSA_ERROR_NOT_SUPPORTED;
 	}
 
-	int si_status = cracen_signature_get_rsa_key(
+	int sx_status = cracen_signature_get_rsa_key(
 		&pubkey, dir == CRACEN_ENCRYPT,
-		psa_get_key_type(attributes) == PSA_KEY_TYPE_RSA_KEY_PAIR, key_buffer,
+		psa_get_key_type(attributes) == PSA_KEY_TYPE_RSA_sKEY_PAIR, key_buffer,
```

Comment:
```suggestion
		psa_get_key_type(attributes) == PSA_KEY_TYPE_RSA_KEY_PAIR, key_buffer,
```

I think this is a stray character that got added